server/store/subscriptions.go

```diff
@@ -7,21 +7,22 @@ import (
 	"github.com/pkg/errors"
 
 	"github.com/mattermost/mattermost-plugin-apps/apps"
-	"github.com/mattermost/mattermost-plugin-apps/server/api"
+	"github.com/mattermost/mattermost-plugin-apps/server/config"
 	"github.com/mattermost/mattermost-plugin-apps/server/utils"
 )
 
-type SubStore struct {
-	*Store
+type Subscription interface {
```

Comment:
2/5 - why not keep the previous SubscriptionStore, AppStore, etc - find it more readable that reusing the entity names for interface names as well.

Reply:
Will change back. Note that the pointers to service interfaces are usually named after the "substance", i.e. `proxy proxy.Service` and then `proxy.Call` etc.

Reply:
@catalintomai is it better now?

Reply:
Thanks @levb, I just find them more readable.